src/sentry/incidents/logic.py

```diff
@@ -591,6 +600,146 @@ class AlertRuleNameAlreadyUsedError(Exception):
 DEFAULT_ALERT_RULE_RESOLUTION = 1
 
 
+def validate_unified_alert_rule(data):
+    """Performs validation on an alert rule's POSTed data
+    This includes ensuring there is either 1 or 2 triggers, which each have actions, and have proper thresholds set.
+    The critical trigger should both alert and resolve 'after' the warning trigger. """
+    triggers = data.get("triggers", [])
+    if triggers:
+        if len(triggers) == 1:
+            if triggers[0]["label"] != CRITICAL_TRIGGER:
```

Comment:
Could you hit a KeyError here if label is not provided? Or will that constraint be enforced earlier in validation?

Reply:
Label is sent by the frontend (not an input field by the user) as "CRITICAL"/"WARNING" atm. I think it is safe to assume proper API usage since it is a private API? Or do you think I should do key checks first?